requirements/install.pip

```diff
@@ -1,2 +1,2 @@
-udata>=1.2.6
+udata>=1.3.0.dev6747
```

Comment:
Managed to fix the card design by specifying the build just before the card redesign PR was merged `udata==1.3.0.dev6411`